### PR TITLE
Add some notes about 3.0 on the download page

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -43,6 +43,12 @@
         conjunction with a FIPS capable version of OpenSSL (1.0.2 series). A
         new FIPS module is currently in development.</p>
 
+        <p>OpenSSL 3.0 is the next major version of OpenSSL that is currently
+        in development and includes the new FIPS Object Module. A pre-release
+        version of this is available below. This is for testing only. It should
+        not be used for security criticial purposes. Information and notes
```
Comment (romen) [inline source/index.html]: There is a typo in critical

Comment (t8m) [reply source/index.html]: I would even suggest `s/for security criticial/in production/`.

Comment (mattcaswell) [reply source/index.html]: Fixed as per @t8m's suggestion

**Comment by mattcaswell** [comment]
Ping...looks like this got forgotten about.

**Review by levitte — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**